ContentstackPersistence/SyncManager.m

```diff
@@ -92,23 +92,34 @@ -(NSString*) getSyncToken {
     return syncToken;
 }
 
+-(NSString*) getSeqId {
+    __block NSString *seqId;
+    [self.persistanceDelegate performBlockAndWait:^{
+        id<SyncStoreProtocol> syncStack  = [self findOrCreate:self->_syncStack predicate:nil];
+        seqId = syncStack.seqId;
+    }];
+    return seqId;
+}
+
 -(void)updateSyncStack:(SyncStack*)syncStack {
     id<SyncStoreProtocol> syncStore = (id<SyncStoreProtocol>)[self findOrCreate:_syncStack predicate:nil];
     syncStore.syncToken = syncStack.syncToken;
     syncStore.paginationToken = syncStack.paginationToken;
+    syncStore.seqId = syncStack.seqId;
 }
 
 -(void)syncWithInit:(BOOL) shouldInit onCompletion:(void (^)(double, BOOL, NSError * _Nullable))completion {
-    NSString *paginationToken = [self getPaginationToken];//csb0e8704474a9624785098d233edd2715`
-    NSString *syncToken = [self getSyncToken];//cse053899d15e9e94cd3751df26f719c87
```

Comment:
commented tokens should be removed

Reply:
Hi @ishaileshmishra , yes they are removed.